### ISSUE TITLE
Wechsel auf PHP8

### ISSUE THREAD

**Issue by hansmorb** [wielebenwir/commonsbooking#1168]
PHP 7.4. ist seit November 2022 EOL, wir sollten unsere composer.json und die dependencies umstellen.

**Comment by hansmorb** [comment]
Da mehr als 60% aller Websites irgendwas unter PHP7.4 verwenden sollten wir das wohl besser nicht machen. 
https://wordpress.org/about/stats/

**Comment by nelarsen** [comment]
Ich wollte cb (master branch) mit "git clone" und "composer install --no-dev" installieren, aber scheinbar wird PHP 8 noch nicht unterstützt? Ist PHP 7.4 Pflicht? Wenn ja, müsste readme.txt geändert werden. Es steht dort "Requires PHP: 7.4 or higher"

(Ich bekam die Fehlermeldung: "ed-itsolutions/cmb2-field-ajax-search v2.0.0 requires php ^5.3.3 || ^7.0 which does not match your installed version 8.1.2")

**Comment by hansmorb** [comment]
Stimmt, in der Doku fehlt, dass du PHP 7.4 für den composer install Befehl nehmen solltest. Aber probier es Mal mit Composer install --no-dev --ignore-platform-reqs

**Comment by hansmorb** [comment]
Ich krame diese issue mal wieder raus, weil es zunehmend schwieriger wird die dependencies so zu resolven, dass PHP 7.4 noch unterstütz wird. [Aktuelle WP Statistik](https://wordpress.org/about/stats/):

<img width="646" height="537" alt="Image" src="https://github.com/user-attachments/assets/70197b9f-5bd9-478c-929a-777cdd2eaf66" />

Ich glaube ein switch auf PHP 8.1 wäre angemessen. PHP 8.0 ist nur ein kleiner slice. PHP 8.1 ist erst seit 2 Monaten EOL.
